src/org/jitsi/util/RTPUtils.java

```diff
@@ -294,4 +294,53 @@ public static boolean isNewerTimestampThan(long a, long b)
     {
         return rtpTimestampDiff(a, b) > 0;
     }
+
+    private static String toHexString(byte b)
+    {
+        char[] hexes
+            = new char[]
```

Comment:
Can we make these static? Also, missing docs

Reply:
fixed